docs/tutorials/12_deuteron_binding_energy.ipynb

```diff
@@ -0,0 +1,1286 @@
+    "                        - np.sqrt((j + 1) * (j + (3 / 2)) * kronecker_delta_function(i - 1, j))\n",
+    "                    )\n",
+    "                    * FermionicOp(label, display_format=\"sparse\")\n",
+    "                )\n",
+    "                hamiltonian += (\n",
+    "                    -5.68658111\n",
+    "                    * kronecker_delta_function(0, j)\n",
+    "                    * kronecker_delta_function(i, j)\n",
+    "                    * FermionicOp(label, display_format=\"sparse\")\n",
+    "                )\n",
+    "\n",
+    "        mapper = JordanWignerMapper()\n",
+    "        qubit_converter = QubitConverter(mapper=mapper)\n",
+    "        qubit_hamiltonian = qubit_converter.convert(hamiltonian)\n",
+    "\n",
+    "        return qubit_hamiltonian\n",
+    "    else:\n",
+    "        raise ValueError(\n",
+    "            \"The function expects an integer number ('int') as input, but was given a {} type.\".format(\n",
+    "                type(N)\n",
+    "            )\n",
+    "        )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b7efbbda",
+   "metadata": {},
+   "source": [
+    "Now, we have an idea of how to use some of Qiskit Nature tools to build a Hamiltonian in terms of Pauli operators. However, this is not the end, we need to construct an ansatz through a parameterized quantum circuit and use it in the VQE to compute the minimum eigenvalue of the Deuteron Hamiltonian (binding energy)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "ea89bb8f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def callback(eval_count, parameters, mean, std):\n",
+    "    # Overwrites the same line when printing\n",
+    "    display(\"Evaluation: {}, Energy: {}, Std: {}\".format(eval_count, mean, std))\n",
+    "    clear_output(wait=True)\n",
+    "    counts.append(eval_count)\n",
+    "    values.append(mean)\n",
+    "    params.append(parameters)\n",
+    "    deviation.append(std)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9872b161",
+   "metadata": {},
+   "source": [
+    "In Dirac notation (bra-ket), the definition of the expectation value of an observable $\\hat{O}$ is equal to [[3]](https://www.feynmanlectures.caltech.edu/III_toc.html) [[4]](https://en.wikipedia.org/wiki/Expectation_value_(quantum_mechanics)):\n",
+    "\n",
+    "$\\langle \\hat{O} \\rangle_\\psi = \\langle \\psi(\\vec{\\theta})|\\hat{O}|\\psi(\\vec{\\theta}) \\rangle$.\n",
+    "\n",
+    "The code below defines a function that calculates the expectation values of observables, given a parameterized quantum circuit and a list with some values for these parameters (angles). To build the `calculate_observables_exp_values` we use some tools of Qiskit Opflow."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "56ba6779",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def calculate_observables_exp_values(\n",
```

Comment:
I'll work on that

Reply:
The new implementation looks a lot better, thanks for that! :+1: 

That said, I only realized now that this method is only used in Step 4 of the tutorial. Maybe we can move the definition of this method into that Step in order to keep things as simple as possible early on in the tutorial? This will also allow you to use the introduction of Step 4 as a motivation for why you need to define this method in the first place